docs/guides/generic-machine-types.md

```diff
@@ -7,7 +7,7 @@ subcategory: Development
 
 The table below is a more detailed version of the common choices summarised in [Task Machine Types](https://registry.terraform.io/providers/iterative/iterative/latest/docs/resources/task#machine-type).
 
-| Type      | [`aws`]       | [`az`]                 | [`gcp`]                                         | [`k8s`]                                              |
+| Type      | [aws]         | [az]                   | [gcp]                                           | [k8s]                                                |
```

Comment:
I still think [Azure Kubernetes Service](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/testing-kubernetes) could fit as a Guide.

And then this page ([Generic Machine Types](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/generic-machine-types)) would be the only "reference" left. I'd just throw it at the bottom of the guides as "Machine Types Reference" since it wouldn't make sense to make a Reference section for a single page. OR can it be listed by itself at the top-level between Guides and Resources?